packages/core/misc-widgets/src/components/github-star/GithubStar.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div
```

Comment:
suggestion: let's only render the content if the url is provided
```suggestion
  <div
    v-if="url"
```

Reply:
Updated 👍